scala3doc/src/dotty/renderers/ScalaSearchbarDataInstaller.scala

```diff
@@ -53,8 +60,11 @@ class ScalaSearchbarDataInstaller(val ctx: DokkaContext) extends SearchbarDataIn
     pages.addOne(p.getName + link, PageEntry(p.getName, p.getName, link, ""))
   }
 
+  private def createAcronym(s: String): Option[String] =
+    s.headOption.filter(_.isUpper).map(_ => s.filter(_.isUpper))
```

Comment:
1) can we move this part to scalaJS? 
2) fuzzt search works with names starting with lowerCase like `lowerCaseName`. `lowerCaseName` will be matched if user provides e.g. `loCN` or "CN" or `lCasNam`

Reply:
Moved creating acronym to JS